tests/test-daemon-integration.py

```diff
@@ -48,7 +48,8 @@ def setUpClass(klass):
         dbus.mainloop.glib.DBusGMainLoop(set_as_default=True)
 
         """Set up a mock system bus."""
-        klass.start_system_bus()
+        klass.start_session_bus()
+        os.environ['DBUS_SYSTEM_BUS_ADDRESS'] = os.environ['DBUS_SESSION_BUS_ADDRESS']
```

Comment:
I suppose you tested this and it works, but I'd imagine a session bus behaves a bit differently than a system bus. I think it would be worth adding a comment here about why this is being done. It looks like in https://github.com/endlessm/endless-sdk-flatpak/commit/bd70f4285a354a5227b955078bf91627d8705311 you're providing a custom dbus system.conf file that allows anonymous authentication. Why doesn't that work here?

Reply:
Oh, yeah, that could definitely use a comment. I added an explanation :)